client/src/hobbies.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './App.css';
+
+
+function Hobbies() {
+    return (
+      <div class="hobbies">
+        <h1>Hobbies Page</h1>
+        <ul>
+          <h2>Watching Sports</h2>
+          <li>Avid UFC watcher</li>
+          <li>Diehard Minnesota Vikings fan</li>
+          <li>Basketball enthusiast</li>
+          <li>Manchester United fan</li>
+          <h2>Playing Video Games</h2>
+          <li>Lifelong gamer who's favorite genres are RPG's, first-person shooters, and action-adventure games</li>
+          <li>Some of my favorite games include: Diablo II: Lord of Destruction, God of War, Batman: Arkham Knight, Assassin's Creed Odyssey</li>
```

Comment:
I love D2, Have you played Grim Dawn? I've been playing through it and it feels incredibly similar to D2 with different stories and classes. But the same type of game.